Source/DafnyCore/DafnyOptions.cs

```diff
@@ -125,6 +125,10 @@ public class DafnyOptions : Bpl.CommandLineOptions {
       }
 
       RegisterLegacyUi(CommonOptionBag.AddCompileSuffix, ParseBoolean, "Compilation options", "compileSuffix");
+
+      RegisterLegacyUi(CommonOptionBag.ReadsClausesOnMethods, ParseBoolean, "Language feature selection", "readsClausesOnMethods", @"
+0 (default) - Reads clauses on methods are forbidden.
+1 - Reads clauses on methods are permitted.".TrimStart(), defaultValue: false);
```

Comment:
This is good, but it might be nice to also mention the `reads *` default, since the value/verbosity ratio is high.

Reply:
Yes I agree on that

Reply:
Good call, I added this to the new CLI help string too.